exercises/news-items/index.html

```diff
@@ -1,6 +1,7 @@
 <html>
 
   <head>
+    <title>News Items</title>
```

Comment:
This is good. But let's change it to News Items Exercise Awesome sauce.

Reply:
Copy that SinJazz Awesome Sauce!

Reply:
Good one 👍